components/scream/src/dynamics/homme/interface/homme_params_mod.F90

```diff
@@ -524,10 +479,43 @@ function get_homme_bool_param_f90 (param_name_c) result(param_value) bind(c)
 
   end function get_homme_bool_param_f90
 
+  function get_homme_nsplit_f90 (atm_dt) result(nsplit_out) bind(c)
+    use control_mod, only: compute_nsplit=>timestep_make_eam_parameters_consistent
+    use homme_context_mod, only: par
+    !
+    ! Input(s)
+    !
+    integer (kind=c_int) :: atm_dt
+    !
+    ! Local(s)
+    !
+    integer (kind=c_int) :: nsplit_out
+    integer :: ierr, nstep_factor
+
+    nsplit_out = -1
+    ierr = compute_nsplit(par, dt_remap_factor, dt_tracer_factor, nsplit_out, nstep_factor, tstep, atm_dt)
```

Comment:
Yeah, unfortunately no. The `dt_xyz_factor` states how many dts are in a xyz step. So `dt_remap_factor` states how many dt's are in a remap, and same for tracer. So those two are fully internal in homme.

We actually had a discussion in the homme slack channel. It seems that the "ideal" scenario would be for atm to simply pass dt_atm to homme, and let homme handle all the subcycling internally (including computing nsplit,rsplit,qsplit, or their "factor" counterparts). That would allow atm to be fully agnostics to the fact that dyn might have certain dt restrictions. But in a way, scream _is_ agnostic to that: it's just HommeDynamics that knows about homme split factors, and it's somewhat semantical the distinction of whether HommeDynamics belongs to scream or Homme.